pkg/sender.go

```diff
@@ -69,6 +69,12 @@ func NewWebRTCSender(ctx context.Context, track *webrtc.Track, sender *webrtc.RT
 }
 
 func (s *WebRTCSender) sendRTP() {
+	// There exists a bug in chrome where setLocalDescription
+	// fails if track RTP arrives before the sfu offer is set.
+	// We deplay sending RTP here to avoid the issue.
+	// https://bugs.chromium.org/p/webrtc/issues/detail?id=10139
+	time.Sleep(500 * time.Millisecond)
```

Comment:
I think this should be long enough 🤷‍♂️

Reply:
I don't have encountered this bug before (and the SFU test server is in another country), so I think that this delay it's long enough to be sure that all the signaling is complete, and don't feel any unnecessary delay in the stream.